charts/ratify/templates/verifier.yaml

```diff
@@ -101,10 +101,11 @@ spec:
             {{- end }}
           {{- end}}
           trustedIdentities:
-            {{- range $j, $store := $policy.trustedIdentities }}
-            {{- if eq $store "*" }}
+            {{- $trustedIdentities := $policy.trustedIdentities | default (list "*") }}
+            {{- range $j, $identity := $trustedIdentities }}
+            {{- if eq $identity "*" }}
```

Comment:
is this fix covered by e2e?

Reply:
I made a change to the second trustPolicy and removed trustedIdentities from input, to ensure that the rendered template will set it to `*`